SoftLayer/managers/firewall.py

```diff
@@ -319,3 +320,20 @@ def get_firewalls_gatewalls(self):
         _filter = {"networkGateways": {"networkFirewall": {"operation": "not null"}}}
 
         return self.account.getNetworkGateways(mask=mask, filter=_filter)
+
+    def get_summary(self, identifier, start_date, end_date):
+        """Returns the metric data for the date range provided
+
+        :param integer metric_tracking_id
+        """
+        body = [{
+            "keyName": "PUBLICIN",
+            "summaryType": "sum"
+
+        }, {
+            "keyName": "PUBLICOUT",
+            "summaryType": "sum"
+        }]
+
+        return self.client['Metric_Tracking_Object'].getSummaryData(start_date,
+                                                                    end_date, body, 1800, id=identifier)
```

Comment:
Since were getting data for a whole month, lets use the max summary period so we get less data from the API
```suggestion
        return self.client['Metric_Tracking_Object'].getSummaryData(start_date,
                                                                    end_date, body, 86400 , id=identifier)
```